keyboards/handwired/neito/config.h

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2020 Oliver Granlund
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x2602
```

Comment:
This vendor ID is already in use. 
https://www.the-sz.com/products/usbid/index.php?v=0x2602&p=&n=

```
❯ echo -n "Oliver Granlund" | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'   
B9F7
```

```suggestion
#define VENDOR_ID       0xB9F7
```
Or if you want to use ascii
```suggestion
#define VENDOR_ID       0x4F47 // "OG"
```